owncloud/ios-app
# [feature/ipad-pointerinteraction] Implementation of iPad UIPointerInteraction to give the user a vi…

## Description
Give the user a visual feedback for some custom buttons on mouse cursor movement.

## Related Issue
#655 

## Motivation and Context
Better UX when using a mouse or trackpad

## How Has This Been Tested?
- Run app on iPad with iOS 13.4
- Connect a mouse or trackpad
- Move the cursor over the elements, mentioned in the issue
- Element should be highlighted

## Types of changes


## Checklist:



### QA

Test plan:

https://github.com/owncloud/QA/blob/master/Mobile/iOS-app/Version%201.4.0/Trackpad%20%26%20Mouse%20support%20iPad.md
